packages/common/components/blocks/content/feed-full.marko

```diff
@@ -78,9 +78,9 @@ $ const textAdButtonLinkStyle = {
             <td valign="top">
               <for|node, index| of=nodes>
                 <if(node.type !== 'text-ad')>
-                  $ const imgWaterMarkOptions = waterMarkDisplay({ node, newsletter, markWidth: 185, markHeight: 185 });
+                  $ const imgWaterMarkOptions = waterMarkDisplay({ node, newsletter, markWidth: 45, markHeight: 45 });
```

Comment:
@Shinsina why is this the only one with 45 as the watermark?  The others in this PR calling the 150 img size are calling 80?